src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -321,14 +317,20 @@ public bool CanNavigateBackground
         public bool NavigationKeyIsDown
         {
             get { return navigationKeyIsDown; }
-            set 
-            { 
+            set
+            {
                 navigationKeyIsDown = value;
                 RaisePropertyChanged("NavigationKeyIsDown");
                 RaisePropertyChanged("CanNavigateBackground");
             }
         }
-        
+
+        public IEffectsManager EffectsManager { get; private set; }
+
+        public IRenderTechniquesManager RenderTechniquesManager { get; private set; }
+
+        public bool SupportDeferredRender { get; private set; }
```

Comment:
Add documentation for new public properties as described in the coding standards.